api-service/cloudify_api/models.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import MetaData
+from sqlalchemy.ext.declarative import declarative_base
+
+from manager_rest.storage.resource_models import AuditLog  # noqa
+
+
+BaseModel = declarative_base(metadata=MetaData(naming_convention={
+    # This is to generate migration scripts with constraint names
```

Comment:
Hopefully all the migrations go in the same place. I'm assuming we can't easily share models?

Reply:
on second thought, perhaps making this depend on restservice is a really really bad idea

Reply:
I know, yagni (at least for some time in the future)…  I guess I was using it in `db_list` which is now gone.  The other argument was the "future self" of the code.  I will get rid of that.

Reply:
I think perhaps making the restservice and this depend on shared DB models would be good though? Otherwise we could easily end up with situations where we have to write the same model in two different places.